### PR TITLE
update/attendee-api

### PR DESCRIPTION
**Pull request by Akalanka47000**
synced with latest backend updates

### CODE REVIEW

File: pages/api/event/event.interface.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ export interface EventData {
   photos: string[]
   tags: string[]
   capacity:number
+  attendeeCount:number
   category: string
   speakers: SpeakerData[]
   attendees:AttendeeData[]
```
Comment (chamodshehanka) [inline pages/api/event/event.interface.ts:24]: @Akalanka47000 are u planning to update the Event record when new users RSVP to the event?

Comment (Akalanka47000) [reply pages/api/event/event.interface.ts:24]: yes

Comment (Akalanka47000) [reply pages/api/event/event.interface.ts:24]: Don't merge this PR though until the latest PR to the backend is confirmed as okay, this is in sync with that

Comment (chamodshehanka) [reply pages/api/event/event.interface.ts:24]: Okay then

**Review by chamodshehanka — APPROVED** [review]

**Pull request merged**